HeatStroke/Graphics/Model/ModelManager.cpp

```diff
@@ -57,8 +57,13 @@ void HeatStroke::ModelManager::Preload(const std::string& p_strPreloadFileName)
 	{
 		std::string strOBJFileName;
 		EasyXML::GetRequiredStringAttribute(pModelElement, "path", strOBJFileName);
-		Model* pModel = new Model(strOBJFileName);
-		m_mLoadedModels[strOBJFileName] = pModel;
+
+		LoadedModels::const_iterator it = m_mLoadedModels.find(strOBJFileName);
+		if (it != m_mLoadedModels.end())
```

Comment:
It should be (it == m_mLoadedModels.end()). You want it to reach the end because you don't want it already in the list.